src/paas_charm/_gunicorn/charm.py

```diff
@@ -23,6 +26,67 @@ def _workload_config(self) -> WorkloadConfig:
             framework_name=self._framework_name, unit_name=self.unit.name
         )
 
+    def check_gevent_package(self) -> bool:
+        """Check that gevent is installed.
+
+        Returns:
+            True if gevent is installed.
+        """
+        ddddd = self._container.exec(["pip", "list"])
```

Comment:
May I know the reason for this variable naming?

Reply:
I was testing first so did not think about a variable name but it turns out I forgot to name it properly before committing. Thank you for noticing!